src/peft/tuners/ia3/layer.py

```diff
@@ -27,12 +27,9 @@
 class IA3Layer(BaseTunerLayer):
     # All names of layers that may contain adapter weights
     adapter_layer_names = ("ia3_l",)
-    # All names of other parameters that may contain adapter-related parameters
-    other_layer_names = ("scaling",)
 
     def __init__(self, base_layer: nn.Module, is_feedforward: bool, **kwargs) -> None:
         self.base_layer = base_layer
-        self.scaling = {}
```

Comment:
As far as I understand, we do not use `self.scaling` and $(IA)^3$ does not have this parameter.